lib/util/theme_utils.dart

```diff
@@ -34,13 +34,15 @@ class ThemeUtils {
     if (isOutline) {
       return TextButton.styleFrom(
         padding: padding,
+        fixedSize: Size(buttonWidth??Size.infinite.width, buttonHeight??Size.infinite.height),
```

Comment:
This looks dangerous - setting infinite size. Did you try on all different permutations?

Reply:
put this Button (without explicit size) inside a Row or Column and it still work?

Reply:
I have tested it without giving height and width in column and row and works fine for me i am testing it with other use cases.